Cargo.toml

```diff
@@ -19,6 +19,21 @@ targets = [
   "x86_64-apple-darwin",
 ]
 
+[package.metadata.packager]
+name = "verso"
+product-name = "verso"
+identifier = "com.crabnebula.verso"
+version = "0.0.1"
+before-each-package-command = "cargo build --release"
+resources = [
+  "resources",
+  "demo.html"
+]
+icons=[
+  "resources/icons/icon256x256.png",
+  "resources/icons/icon.ico"
```

Comment:
Where do these logos come from? Could we skip them as well?

Reply:
These are prototype logo. We had some discussion over discord and I picked one of the logos for now. As I mentioned in PR, this is not final.

Also, no, we cannot skip this as a **square icon** is a must for `AppImage` bundle.